src/ClientData/include/ClientData/FunctionUtils.h

```diff
@@ -34,7 +34,11 @@ namespace function_utils {
 [[nodiscard]] uint64_t Offset(const orbit_client_protos::FunctionInfo& func,
                               const ModuleData& module);
 
-[[nodiscard]] std::optional<uint64_t> GetAbsoluteAddress(
+// This function should not be used, since it could return incomplete or invalid
+// result in the case when one module is mapped 2 or more times. Try to find another way
```

Comment:
```suggestion
// result in the case when one module is mapped two or more times. Try to find another way
```
Wrote this on one of Anton's recent changes, I feel like I should do the same here :)

Reply:
done.